make/migrations/postgresql/0140_2.11.0_schema.up.sql

```diff
@@ -28,4 +28,15 @@ then set column artifact_type as not null
 */
 UPDATE artifact SET artifact_type = media_type WHERE artifact_type IS NULL;
 
-ALTER TABLE artifact ALTER COLUMN artifact_type SET NOT NULL;
\ No newline at end of file
+ALTER TABLE artifact ALTER COLUMN artifact_type SET NOT NULL;
+
+CREATE TABLE IF NOT EXISTS sbom_report
+(
+    id SERIAL PRIMARY KEY NOT NULL,
+    uuid VARCHAR(64) UNIQUE NOT NULL,
+    artifact_id INT NOT NULL,
+    registration_uuid VARCHAR(64) NOT NULL,
+    mime_type VARCHAR(256) NOT NULL,
+    report JSON,
```

Comment:
Does the `report JSON` mean the SBOM report content defined in pluggable-scanner-spec v1.2? In that case, the report contains SBOM raw data, which could be too large to be stored in database.

Reply:
Not the original report content, it just store a the summary information such as the digest, repository etc